collectors/ebpf.plugin/ebpf_sync.h

```diff
@@ -46,7 +46,7 @@ enum netdata_sync_charts {
 };
 
 enum netdata_sync_table {
-    NETDATA_SYNC_GLOBLAL_TABLE
+    NETDATA_SYNC_GLOBAL_TABLE
```

Comment:
Hopefully this is private and only needed by the preceding file

Reply:
Yes, you are right, this won't be shared with any other thread. The `GLOBAL` here, it is because it affects 6 different eBPF programs, and these eBPF programs are separated files on other repo.

I copied the name to keep correlation between repositories.